xAPI.md

```diff
@@ -3226,10 +3226,21 @@ include attachment raw data and MUST report application/json.
 
 ###### Filter Conditions for StatementRefs 
 
-For filter parameters which are not time or sequence based (that is, other than
-since, until, or limit), Statements which target another Statement (by using a StatementRef
+This section outlines rules by which Statements targeting other Statements can sometimes be considered to 
+meet the filter conditions of a query even if they do not match the original query's filter parameters. 
+These rules **do not** apply when retrieving a single Statement using "statementId" or "voidedStatementId" query 
+parameters.
+
+'Targeting Statements' means that one Statement (the targeting Statement) includes the Statement Id of another
+Statement (the targeted Statement) as a Statemnent Reference either as the object of the Statement or within the 
+"statement" property of the Context. 
```

Comment:
This seems to be an expansion, taken with some parts below. In the next sentence you see that it only applied (applies?) to ones in the object of the statement, not ones in context... but then later on line 3284 we refer to the generalized targeting definition, not the definition restricted to just the object. I believe we need to remove the context mention.